src/features/Dashboard/InstructorAssignSection/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { Row, Col } from '@edx/paragon';
+import ClassCard from 'features/Dashboard/InstructorAssignSection/ClassCard';
+import { Button } from 'react-paragon-topaz';
+
+import { fetchClassesData } from 'features/Dashboard/data';
+
+import 'features/Dashboard/InstructorAssignSection/index.scss';
+
+const InstructorAssignSection = () => {
+  const dispatch = useDispatch();
+  const stateInstitution = useSelector((state) => state.main.institution.data);
+  const classesData = useSelector((state) => state.dashboard.classes.data);
+  const [classCards, setClassCards] = useState([]);
+  let idInstitution = '';
+  // eslint-disable-next-line no-unused-expressions
+  stateInstitution.length > 0 ? idInstitution = stateInstitution[0].id : idInstitution = '';
+
+  useEffect(() => {
+    dispatch(fetchClassesData(idInstitution));
+  }, [idInstitution]); // eslint-disable-line react-hooks/exhaustive-deps
+
+  useEffect(() => {
+    // Show in home only two first classes
+    if (classesData.length > 2) {
```

Comment:
What do you think set the number 2 as a variable? @AuraAlba 
```suggestion
    if (classesData.length > NumberOfClasses) {
```

Reply:
Done